samples/app-complete-sample/nodejs/README.md

```diff
@@ -38,6 +38,20 @@ Please find below demo manifest which is deployed on Microsoft Azure and you can
 
 - [NodeJS](https://nodejs.org/en/)
 - [ngrok](https://ngrok.com/) or equivalent tunnelling solution
+- [Teams Toolkit for VS Code](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) or [TeamsFx CLI](https://learn.microsoft.com/microsoftteams/platform/toolkit/teamsfx-cli?pivots=version-one)
+
+## Run the app (Using Teams Toolkit for Visual Studio Code)
```

Comment:
I see this sample requires 'Facebook Authentication'. How does it work by using TTK?

Reply:
Suggest having a check whether it is possible to update azure.bicep file to automate the creation of Facebook connection

Reply:
We mapped ConnectionName = OAuth connect, its working as per readme. 
![MicrosoftTeams-image (22)](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/117716554/b7d8fc45-0060-4e89-bd2a-a0637c270ee3)
![MicrosoftTeams-image (23)](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/117716554/7cd83ecd-2c9b-4092-a0b5-5723856c0d90)
![MicrosoftTeams-image (24)](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/117716554/a9650c72-aaa1-417d-8154-4da85f1c0a8e)

Reply:
So you need to add those manual steps under this section `Run the app (Using Teams Toolkit for Visual Studio Code)`. Otherwise, user has n context about this. he did not know that he need to create Fackbook things manually.

Reply:
Fixed. 
![image](https://github.com/OfficeDev/Microsoft-Teams-Samples/assets/117716554/5b3cc858-5082-4a84-896f-922216b3325e)